### PR TITLE
Up down key shortcut

### LINKED ISSUE #100: Shortcut to move up and down previously entered orders.

### PR DESCRIPTION
**Pull request by pavz02**

### CODE REVIEW

File: src/main/java/seedu/cakecollate/ui/CommandBox.java [hunk 1]
```diff
@@ -69,6 +75,68 @@ private void setStyleToIndicateCommandFailure() {
...
+    }
+
+    private void decrementUserInputsIndex() {
+        if (!firstDecrementAfterUserInput && userInputsIndex > 0) {
+            userInputsIndex--;
+        }
+        firstDecrementAfterUserInput = false;
+    }
+
+    public void setCommandTextField(Optional<String> input) {
```
Comment (simran-bhadani3) [inline src/main/java/seedu/cakecollate/ui/CommandBox.java]: Remember to add Javadoc comments for this method and also the other public methods,

File: src/main/java/seedu/cakecollate/ui/CommandBox.java [hunk 2]
```diff
@@ -69,6 +75,68 @@ private void setStyleToIndicateCommandFailure() {
...
+
+    public TextField getCommandTextField() {
+        return commandTextField;
+    }
+
+    public String getTextInCommandTextField() {
+        return commandTextField.getText();
+    }
+
     /**
```
Comment (simran-bhadani3) [inline src/main/java/seedu/cakecollate/ui/CommandBox.java:184]: There seems to be a stray Javadoc comment here.

**Review by simran-bhadani3 — COMMENTED** [review]
Overall looks pretty good!

File: src/main/java/seedu/cakecollate/ui/CommandBox.java (hunk 2 above)
Comment (pavz02) [reply src/main/java/seedu/cakecollate/ui/CommandBox.java:184]: There's a method underneath haha

Comment (simran-bhadani3) [reply src/main/java/seedu/cakecollate/ui/CommandBox.java:184]: Oh XD

**Pull request merged**